fix primary color codes (#48)
rsksmart/devportal: rif/rns/operation/Register-a-name.md

title: Register a name
---

The process consists of three main steps:

1. [Check availability](#check-availability)
2. [Request to register](#request-to-register)
3. [Reveal commit to register the domain](#reveal-commit-to-register-the-domain)

## Check availability

You need to check if your name is available. Go to the <a href="https://beta.manager.rns.rifos.org/search">RNS Manager</a> and check it!

> If your name is available, click on "register your domain" and go to the next step.

Reviewer: ```suggestion
If your name is available, click on "register your domain", and go to the next step.
```

<img src="/assets/img/rns/register-step-1.png" class="img-fluid" alt="step-1" />

## Request to register

> Select for how long you want to register your name, check that you have that amount of RIF Tokens and click on "Request to register".

Reviewer: ```suggestion
Select the duration for which you would like to register your name, check that you have that amount of RIF Tokens, and click on "Request to register".
```

Sign the transaction and wait for one minute. We're almost done!

<img src="/assets/img/rns/register-step-2.png" class="img-fluid" alt="step-2" />

## Reveal commit to register the domain

> Once you've waited for aprox. one minute, the "Register" button will be enabled and you will be able to finalize the process.

Reviewer: ```suggestion
After approximately one minute, the "Register" button will be enabled, and you will be able to finalize the process.
```

> Just need to click on it, sign the transaction and you'll get your name!

Reviewer: ```suggestion
To do so, click on the "Register" button, sign the transaction, and you'll get your selected name!
```
